src/llvmir2hll/obtainer/call_info_obtainer.cpp

```diff
@@ -436,7 +437,7 @@ void CallInfoObtainer::SCCComputer::visit(ShPtr<CG::CalledFuncs> calledFunc,
 		// function, do this only if it calls itself (see the description of
 		// computeSCCs()).
 		if (scc.size() != 1 || hasItem(calledFunc->callees, calledFunc->caller)) {
-			sccs.insert(scc);
+			sccs.push_back(scc);
```

Comment:
Here, we could mention that Tarjan's algorithm never creates the same SCC twice, so it is safe to use just `push_back()` without any existence checks. What do you think?

Reply:
Yup. I'm going to probably rewrite this code a bit later, but it's always good to have accurate comments at all times. Fixed